libraries/plugins/xfpga/sysfs.c

```diff
@@ -1690,10 +1690,13 @@ fpga_result get_port_sysfs(fpga_handle handle, char *sysfs_port)
 	return FPGA_OK;
 }
 
-enum fpga_hw_type opae_id_to_hw_type(uint16_t vendor_id, uint16_t device_id)
+enum fpga_hw_type opae_id_to_hw_type(uint16_t vendor_id, uint16_t device_id,
+	uint16_t sub_vendor_id, uint16_t sub_device_id)
 {
 	enum fpga_hw_type hw_type = FPGA_HW_UNKNOWN;
 
+	UNUSED_PARAM(sub_vendor_id);
```

Comment:
Probably should be using sub_vendor_id in the comparison, just for good practice.   They may be all Intel devices now, but they may not be in the future.